lib/diagnostic.js

```diff
@@ -22,13 +30,26 @@ const uniqueWords = function (/* */) {
 // a second argument.
 
 const wordCount = function (/* */) {
-  /* your response here */
+  const i
```

Comment:
You don't need to define `i` outside of the for loop, and in fact it is better to define it inside the loop using the `let` keyword